src/modules/feedback/components/FeedbackModal.js

```diff
@@ -57,34 +57,44 @@ type PropsType = {|
   theme: ThemeType
 |}
 
+type SendingStatusType = 'IDLE' | 'SUCCESS' | 'ERROR'
+
 type StateType = {|
-  feedbackSent: boolean
+  feedbackSent: SendingStatusType
```

Comment:
`feedbackStatus` is of type `FeedbackRatingType`. I didn't change this.

Reply:
But you should change it :smile: Before it was just a boolean which stated whether one had clicked on send. Now, it is more like a status since there are more values.

Reply:
I normally wouldn't change the nomenclature without an agreement, unless it would be an assignment. Please confirm, that this is an assignment. I would then suggest `feedbackRating` for the rating type and `sendingStatus` (as in the NATIVE-App) for the status of sending.

Reply:
You don't need an assignment to do something like this. If a variable, class or method name does not make sense anymore, it should be changed.